worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -78,6 +90,13 @@ public Workflow get(final String workflowName){
 
                 final StringBuilder stringBuilder = new StringBuilder();
                 stringBuilder.append(String.format("var ACTIONS = %s;\n", gson.toJson(workflow.getActions())));
+
+                if (contextScriptFile != null && contextScriptFile.exists()) {
+                    LOG.warn("Context script file from the path " + contextScriptFilePath + " not found.");
```

Comment:
For goodness sake you are warning that the file is not found immediately inside the code block where you have confirmed that is exists.